src/tests/constructors.test.ts

```diff
@@ -110,7 +116,7 @@ describe('fromSuccess', () => {
     const a = composable(() => 1)
 
     const c = fromSuccess(a)
-    type _R = Expect<Equal<typeof c, () => Promise<number>>>
+    type _R = Expect<Equal<typeof c, () => Promise<1>>>
```

Comment:
@diogob I don't know why it changed.. narrower seems better. Didn't break any other test... any ideas?

Reply:
I don't understand, but LGTM